@here/harp-webtile-datasource/lib/WebTileDataSource.ts

```diff
@@ -7,14 +7,20 @@
 import * as THREE from "three";
 
 import { TileKey, TilingScheme, webMercatorTilingScheme } from "@here/harp-geoutils";
-import { CopyrightInfo, DataSource, Tile, UrlCopyrightProvider } from "@here/harp-mapview";
+import {
+    CopyrightInfo,
+    DataSource,
+    TextureLoader,
+    Tile,
+    UrlCopyrightProvider
+} from "@here/harp-mapview";
 import { TileGeometryCreator } from "@here/harp-mapview/lib/geometry/TileGeometryCreator";
 import { getOptionValue, LoggerManager } from "@here/harp-utils";
 
 const logger = LoggerManager.instance.create("MapView");
 
-const textureLoader = new THREE.TextureLoader();
-textureLoader.crossOrigin = ""; // empty assignment required to support CORS
+const textureLoader = new TextureLoader();
+textureLoader.setCrossOrigin(""); // empty assignment required to support CORS
```

Comment:
can this be done in the constructor of the TextureLoader? I don't see the need to do it here...?